tests/actions.rs

```diff
@@ -0,0 +1,235 @@
+//! Alert tests
+#[macro_use]
+extern crate serial_test;
+extern crate fantoccini;
```

Comment:
This shouldn't be necessary?

Reply:
Looks like the first two lines are needed but not the `extern crate fantoccini;`